packages/tasks/src/model-libraries.ts

```diff
@@ -843,11 +843,21 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://github.com/jasonppy/VoiceCraft",
 		snippets: snippets.voicecraft,
 	},
+	yolov8: {
+		prettyLabel: "YOLOv8",
```

Comment:
Shouldn't it be "ultralytics"?

Reply:
I changed it, and left a comment on snippet. there's no way of knowing which yolo version would it be because there's no config file.

Reply:
hmm, if it's going to be generic then I'd prefer `yolo` without a version number.

Reply:
I agree, changed to that. not many people know ultralytics as the brand but they know yolo indeed

Reply:
if it's ok for all of us I can merge

Reply:
Hmm ok, but the library is called `ultralytics` right? People need to do `pip install ultralytics`. So would be in favor of having `library_name: ultralytics`.

For yolov10, we called it library_name: yolov10 since that is also the name of the "library": https://github.com/THU-MIG/yolov10?tab=readme-ov-file#installation.

Reply:
@NielsRogge tbh I think yolo is better for visibility. I doubt that people would immediately recognize ultralytics as the library

Reply:
That's a good point @NielsRogge, but I agree with @merveenoyan here, I think "yolo" is a well-known brand, and very appropriate for an umbrella term :)

Also, checking the snippet from the page you posted I see `from ultralytics import YOLOv10` as well.

Reply:
There are 296 models tagged on the hub with `library_name: ultralytics`: https://huggingface.co/models?other=ultralytics, so this could also be created as a filter on the left. This would also be the main topic of the potential partnership with Ultralytics.

In my opinion, YOLOv8 and YOLOv10 are both models, not libraries. Other libraries can also provide YOLO models, like timm in the near future.

Reply:
I'm fine either way, happy to follow @NielsRogge, especially if you're in contact with them and know what they prefer :)

Reply:
Btw another reason is that otherwise we would associate each yolo model on the hub with a .pt file.

Now we associate Ultralytics with .pt files, which more sense to me, as they use pickle for serialization of weights.

Reply:
> I changed it, and left a comment on snippet. there's no way of knowing which yolo version would it be because there's no config file.


What do you think about using the model tags to know which yolo version to use? I also agree with @NielsRogge here that `ultralytics` feels like the correct library_name here as _it is_ the library used to load the model. I opened a PR to showcase what I have in mind: https://github.com/huggingface/huggingface.js/pull/1101